### ISSUE TITLE
Use kibitzer book with arrow keys

### ISSUE THREAD

**Issue by futurelauncher** [lukasmonk/lucaschessR#78]
Hi,
Kibitzer book can be used with the help of mouse, not keyboard. It would be very comfortable to go through various lines of book with arrow keys.

**Comment by lukasmonk** [comment]
Yes. All kibiters have the same problem. I will work on it. It has now the number 28 in my notebook.
